includes/forms/class-llms-form-templates.php

```diff
@@ -179,6 +231,37 @@ private static function get_confirm_group_controller( $block ) {
 
 	}
 
+	/**
+	 * Retrieves legacy options value for a given field and location
+	 *
+	 * @since [version]
+	 *
+	 * @param string $field_id The field's identifier as found in the block schema list returned by LLMS_Form_Templates::get_reusable_block_schema().
+	 * @param string $location Form location. Accepts "checkout", "registration", or "account".
+	 * @return string
+	 */
+	private static function get_legacy_options( $field_id, $location ) {
```

Comment:
```suggestion
	private static function get_legacy_option( $field_id, $location ) {
```

I think this must be singular